### PR TITLE
Remove programming paradigms from main advantages list

### PR DESCRIPTION
**Pull request by histlib**

### CODE REVIEW

**Comment by cmacdonell** [comment]
I might be able to guess, but can you explain why you removed that bullet?

**Comment by histlib** [comment]
Assuming that the lesson is for novices, it seemed like an unnecessarily advanced and potentially confusing point (do you want me to add this to the pull request description?)

**Comment by cmacdonell** [comment]
Yes, please add it. It's always good to have a reason in a PR.

File: _episodes/00-short-introduction-to-Python.md [hunk 1]
```diff
@@ -24,7 +24,6 @@ Python's main advantages:
 * Available on all platforms
 * "Batteries Included" philosophy - libraries for common tasks available in
   standard installation
-* Supports multiple programming paradigms
```
Comment (histlib) [inline _episodes/00-short-introduction-to-Python.md:27]: If the lesson is for novices, then this point is too advanced and too likely to raise unnecessary questions/confusion.

**Comment by histlib** [comment]
I couldn't find a way to edit my pull request comments, so I added a line comment to the pull request. This might not be what you had in mind, but I didn't see another way to do it.

File: _episodes/00-short-introduction-to-Python.md (hunk 1 above)
Comment (cmacdonell) [reply _episodes/00-short-introduction-to-Python.md:27]: 👍

**Pull request merged**